client/interface.go

```diff
@@ -25,9 +25,24 @@ import (
 // CommonAPIClient defines common methods of api client.
 type CommonAPIClient interface {
 	PreheatAPIClient
+	PeerAPIClient
+	TaskAPIClient
 }
 
 // PreheatAPIClient defines methods of Container client.
 type PreheatAPIClient interface {
-	PreheatCreate(ctx context.Context, config *types.PreheatCreateRequest) (*types.PreheatCreateResponse, error)
+	PreheatCreate(ctx context.Context, request *types.PreheatCreateRequest) (*types.PreheatCreateResponse, error)
+}
+
+// PeerAPIClient defines methods of peer related client.
+type PeerAPIClient interface {
+	PeerCreate(ctx context.Context, request *types.PeerCreateRequest) (*types.PeerCreateResponse, error)
```

Comment:
I think we should use the named variables for interface methods.

Reply:
Done